dashboard/v1.1/src/pages/Dashboard/SystemMetrics.tsx

```diff
@@ -141,7 +148,7 @@ const SystemMetrics: FC<SystemMetricsProps> = ({ showLoader }) => {
         }
       );
     // eslint-disable-next-line
```

Comment:
This is valid/required anymore?

Reply:
@Harkishen-Singh are you talking about showLoader props?

Reply:
About the `// eslint-disable-next-line`. Do we still need to ignore them?

Reply:
Yes